content/04.study.md

```diff
@@ -79,60 +79,64 @@ Moreover, these nonlinear algorithms can deconvolve the effects of multiple vari
 The ultimate goal is to predict an individual’s tissue-specific, exon-specific splicing patterns from their genome sequence and other measurements to enable a new branch of precision diagnostics that also stratifies patients and suggests targeted therapies to correct splicing defects.
 However, to achieve this we expect that methods to interpret the "black box" of deep neural networks and integrate diverse data sources will be required.
 
-### Transcription factors and RNA-binding proteins
-
-Transcription factors and RNA-binding proteins are key components in gene regulation and higher-level biological processes.
-TFs are regulatory proteins that bind to certain genomic loci and control the rate of mRNA production.
-While high-throughput sequencing techniques such as chromatin immunoprecipitation and massively parallel DNA sequencing (ChIP-seq) have been able to accurately identify targets for TFs, these experiments are both time consuming and expensive.
-Thus, there is a need to computationally predict binding sites and understand binding specificities *de novo* from sequence data.
-In this section we focus on TFs, with the understanding that deep learning methods for TFs are similar to those for RNA-binding proteins, though RNA-specific models do exist [@doi:10.1186/s12859-017-1561-8].
-
-ChIP-seq and related technologies are able to identify highly likely binding sites for a certain TF, and databases such as ENCODE [@tag:Consortium2012_encode] have made freely available ChIP-seq data for hundreds of different TFs across many laboratories.
-In order to computationally predict transcription factor binding sites (TFBSs) on a DNA sequence, researchers initially used consensus sequences and position weight matrices to match against a test sequence [@tag:Stormo2000_dna].
-Simple neural network classifiers were then proposed to differentiate positive and negative binding sites but did not show meaningful improvements over the weight matrix matching methods [@tag:Horton1992_assessment].
-Later, support vector machines (SVMs) outperformed the generative methods by using k-mer features [@tag:Ghandi2014_enhanced; @tag:Setty2015_seqgl], but string kernel-based SVM systems are limited by their expensive computational cost, which is proportional to the number of training and testing sequences.
```

Comment:
`tag:Setty2015_seqgl` was replaced with `doi:10.1371/journal.pcbi.1000916`.  Does that seem okay?